backend/fms_core/tests/test_services/id_generator_test.py

```diff
@@ -2,7 +2,7 @@
 
 from fms_core.services.id_generator import get_unique_id
 
-class IndividualServicesTestCase(TestCase):
+class IdGeneratorServicesTestCase(TestCase):
```

Comment:
You may want to test for integer overflow ;)

Reply:
If we ever run out of ID we'll have other problems before.